VideoAPI/Testing.Common/Helper/ApiUriFactory.cs

```diff
@@ -96,5 +96,7 @@ public class MessageEndpoints
         private string ApiRoot => "conferences";
         public string GetMessages(Guid conferenceId) => $"{ApiRoot}/{conferenceId}/messages";
         public string SaveMessage(Guid conferenceId) => $"{ApiRoot}/{conferenceId}/messages";
+        public string RemoveMessagesForConference(Guid conferenceId) =>
+           $"{ApiRoot}/{conferenceId}/messages";
```

Comment:
rename to instant message please. There was a discussion this morning and I don't think the AC has been updated by the looks of it

Reply:
Renamed